src/util-profiling.c

```diff
@@ -1284,57 +1326,57 @@ const char * PacketProfileDetectIdToString(PacketProfileDetectId id)
  *
  * \retval string equivalent for the LoggerId id
  */
-const char * PacketProfileLoggertIdToString(LoggerId id)
+const char *PacketProfileLoggertIdToString(LoggerId id)
```

Comment:
Could we restrict the clang-format to the changes in the file and not the whole file ?

That should be something like 
- `git add`
- `git clang-format`
- `git commit` ie commits without the clang-format changes
- `git commit -a` ie commits the clang-format changes

Reply:
hi, that was even the normal behaviour of git-clang-format to format only the modified lines by the commit.
There was a comment a few PR's back to fix these changes because they looked differently than the "surrounding code".

The only solution I saw to make the pipeline run and resolve the comment was to clang-format the whole file :)

Reply:
Fair enough for me :-)